manke/htcondor/htcondor-central-manager-service.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    metallb.universe.tf/address-pool: htcondor
+  labels:
+    name: htcondor-central-manager
+    visualize: "true"
+  name: htcondor-central-manager
+  namespace: htcondor
+spec:
+  ports:
+    - port: 9618
+  selector:
+    app: htcondor-central-manager
+  type: LoadBalancer
+  loadBalancerIP: 139.229.152.211
```

Comment:
`.spec.loadBalancerIP` is deprecated.  As the `htdcondor` pool is a /32, this line could simply be removed or it could be replaced with a `metallb.universe.tf/loadBalancerIPs` annotation.  I would slightly prefer for the former option.  https://metallb.universe.tf/usage/#requesting-specific-ips